### ISSUE TITLE
Check the current version vs the last published version and recommend an upgrade

### ISSUE THREAD

**Issue by dvidsilva** [gloriajs/gloria#70]
I've seen some packages show a message like:

```
You're running version 0.0.1 - our latest version is 0.1.0, run 
npm install -g gloria 
to get access to our latest features
```

Commands like serve don't change much over time, but with commands like build and init, and our fast release cycles previous users might be missing out in features.

**Comment by dvidsilva** [comment]
this package can be used to compare semver https://www.npmjs.com/package/semver

**Comment by dvidsilva** [comment]
Closed on #94
